topics/admin/tutorials/poll-ssa/slides.html

```diff
@@ -0,0 +1,433 @@
+---
+layout: tutorial_slides
+logo: assets/images/gat.png
+video: true
```

Comment:
Removed the video option (for now at least) since the document lacks the required captions. But having added alt-texts to the images with you, hopefully it will still be accessible : https://github.com/galaxyproject/training-material/pull/4972/commits/6b6c0e03b739b8056b597afeac72354940ba1961

Reply:
I am curious if there is a behaviour where the video system
- for each slide, detects if there are captions (the ??? section)
- if not, it uses the alt text of images, as well as the "normal text", as "back-up" captions?
- maybe captioners could just add captions to the few slides that are particularly confusing, instead of to each slide?

Reply:
It's an interesting suggestion @vladvisan, to take slide captions from an alt text. We would have to think about how best to do that, since the purposes are similar, but slightly different: alt text needs to be textually equivalent to the image, the user should learn the entire contents of the image just by reading the text (so slightly more akin to "audio description"), and in captions you can say whatever you like there as there is an implicit assumption the viewer is sighted and can understand the contents of the figure without it being explained in detail to them.

There's probably some slides where that would save the captioner time, it's true! I'll investigate it when I can :)

Reply:
Ah, good point about the hypothesis that the viewer is sighted.
My initial assumption was that the video was for people that were not sighted, but I now realise that isn't the case.
Someone who can't see would probably have a browser that reads the alt-texts and the text for him, rather than playing the video.

Reply:
Also to criticise my own idea: if the caption for some slides really is as simple as "copy and paste the existing text", then it only takes the captioner 3 seconds to copy&paste it to be fair...